index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Dyary's Portfolio</title>
+        <link rel="stylesheet" href="styles.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="container">
+            <header>
+                <ul>
+                    <li id="my-logo-nav-item"><a href="#logo">My Logo</a></li>
+                    <li class="left-nav-items">Portfolio</li>
+                    <li class="left-nav-items">about</li>
+                    <li class="left-nav-items">contact</li>
+                    <li id="hamburger-nav-item">
+                        <img src="images/hamburger-icon.png" alt="hamburger
+                            icon">
+                    </li>
+                </ul>
+            </header>
```

Comment:
Inside your `header` tag, you should have a `nav` tag, and inside the `nav` tag, you should add all that is inside your `ul` tag.

```
<header>
  <nav>
    <ul>
       *** Your list ***
    </ul>
  </nav>
</header>
```

- [ ] Add `nav` tag in your `header` tag.